protocol/ws/v2/subprotocols.go

```diff
@@ -0,0 +1,22 @@
+package v2
+
+import (
+	"fmt"
+
+	"nhooyr.io/websocket"
+
+	"github.com/cloudevents/sdk-go/v2/binding/format"
+)
+
+const JsonSubprotocol = "cloudevents.json"
+
+var SupportedSubprotocols = []string{JsonSubprotocol}
+
+func resolveFormat(subprotocol string) (format.Format, websocket.MessageType, error) {
```

Comment:
The `format` has already supported different format types, are you planing to support different types? I am wondering if we can leverage or improve the existing framework. 
https://github.com/cloudevents/sdk-go/blob/7bec7ba0a7bbc90a67a5fc594a217ee0b01c1bf7/v2/binding/format/format.go#L66

Reply:
In sdk-go we support only the EventFormat atm, so here i'm just mapping to what we support currently

Reply:
Ah I see your point now, well this "mapping" of subprotocols is very specific to websockets protocol binding https://github.com/cloudevents/spec/blob/master/websockets-protocol-binding.md#15-cloudevents-subprotocols, i guess this code should live only in the websockets protocol. Also because it uses a websocket type (`websocket.MessageType`)